lib/index.js

```diff
@@ -94,22 +96,22 @@ export async function retryable(retryableFunc, {maxWaitMs=1000}={}) {
       error = e;
     }
 
-    const elapsedTime = (new Date()).getTime() - startTimestamp;
+    const elapsedTime = new Date().getTime() - startTimestamp;
     if (elapsedTime > maxWaitMs) {
-      const errorMessage = (
+      const errorMessage =
         `Retryable did not complete in allocated time:\n${error.stack}\n\n` +
-        `Failing function:\n${retryableFunc.toString()}\n`
-      );
+        `Failing function:\n${retryableFunc.toString()}\n`;
       console.error(errorMessage);
       throw new Error(errorMessage);
     }
     await sleep(25);
   } while (true); // eslint-disable-line
 }
 
-export async function condition(predicate, {maxWaitMs=1000}={}) {
-  const startTimestamp = (new Date()).getTime();
-  while (true) { // eslint-disable-line
+export async function condition(predicate, {maxWaitMs = 1000} = {}) {
+  const startTimestamp = new Date().getTime();
+  // eslint-disable-next-line
```

Comment:
since you're updating this anyway, can you add the specific rule being disabled rather blanket-disabling everything?